TestAssets/TestProjects/TestLibraryWithMultipleFrameworks/Program.cs

```diff
@@ -7,6 +8,10 @@ public class Program
         public static void Main()
         {
             Console.WriteLine("Hello World!");
+#if !NETSTANDARDAPP1_5
```

Comment:
This is only in a test, but yes it would be affected by a change to 1.6. However, it would be the smallest of a whole host of things that would be affected :).

I can change it to `NET20 || NET35 || NET40 || ...` which will be slightly more stable.